### ISSUE TITLE
qtvstools v3.2: 'DWORD FormatMessageA(DWORD,LPCVOID,DWORD,DWORD,LPSTR,DWORD,va_list *)': cannot convert argument 5 from 'LPWSTR' to 'LPSTR' at custom code in MSVC2019

### ISSUE THREAD

**Issue by Catty82** [qtproject/qt-labs-vstools#31]
**Describe the bug**
Compile error in my Lib.cpp when qtvstools 3.2 is installed.
`Error    C2664    'DWORD FormatMessageA(DWORD,LPCVOID,DWORD,DWORD,LPSTR,DWORD,va_list *)': cannot convert argument 5 from 'LPWSTR' to 'LPSTR'    MyLib    in MyLib.cpp`
```
	LPWSTR bufPtr = nullptr;
	FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
		0, code, 0, (LPWSTR)&bufPtr, 0, 0);

```
This happens in version 3.2, but in version 3.0.2 is compiled well.

**Desktop (please complete the following information):**
 - Qt VS Tool Version [3.2]
 - Visual Studio version [VS2019]

**Comment by acdemiralp** [comment]
Hello,

Are you sure that this problem is not related to other code?

Typically Windows A (ASCII) and W (Wide) functions are selected based on the project's character set, i.e. the value of `Project Properties > Configuration Properties > Advanced > Character Set`:
- If set to `Use Multi-Byte Character Set`, `FormatMessage` macro dispatches to `FormatMessageA` which accepts a `LPSTR`.
- If set to `Use Unicode Character Set`, `FormatMessage` macro dispatches to `FormatMessageW` which accepts a `LPWSTR`.

Projects created via Qt VS Tools wizards (as well as standard VS wizards) will be set to `Use Unicode Character Set`. Can you please check the `Character Set` of your project and set it to `Use Unicode Character Set` if it is already not set to that?

Alternatively, you can use the ASCII and Wide functions directly:
```cpp
LPSTR buffer;
FormatMessageA(..., buffer, ...);
```
or:
```cpp
LPWSTR buffer;
FormatMessageW(..., buffer, ...);
```

**Comment by Catty82** [comment]
I have Not set in project and it works on vstools 3.0.2.
![image](https://github.com/user-attachments/assets/25b4bb12-7d9b-4afc-ad18-3c50b8147659)

But UNICODE has been fixed in 3.1
![image](https://github.com/user-attachments/assets/2684beaf-2f84-45e0-b4c2-acec6427b7b9)
After changing Not Set to Unicode it works on vstools 3.2.

**Comment by Catty82** [comment]
solved
